lib/main.js

```diff
@@ -55,6 +55,11 @@ function bind(context, fn) {
   };
 }
 
+// Define textContent method to safely fetch textContent of unknown elements
```

Comment:
Grammar fix here: "textContent of" should be "textContent from". One doesn't really "fetch a ball of the ground", instead you "fetch a ball from the ground".